src/include/daos_cont.h

```diff
@@ -12,7 +12,9 @@
 #ifndef __DAOS_CONT_H__
 #define __DAOS_CONT_H__
 
+/** container open API macro to invoke latest implementation */
 #define daos_cont_open daos_cont_open2
+/** container destroy API macro to invoke latest implementation */
 #define daos_cont_destroy daos_cont_destroy2
```

Comment:
we should probably just add something like:
Please ignore (code for backward interoperability)